generation/core/type-checks.js

```diff
@@ -26,8 +26,33 @@ const isGreyMatcher = ({ name }) =>
     throw new Error('${name} should be a GREYMatcher, but got ' + JSON.stringify(ARG));
   }
 `)({
-		ARG: t.identifier(name)
-	});
+			ARG: t.identifier(name)
+		});
+const isGreyAction = ({ name }) =>
```

Comment:
What do you think about just defining functions instead of assigning functions to variables ?

```js
function isGreyAction({name}) {
}
```

Reply:
I like the idea 👍